Imports/ArcGIS/Runtime/Toolkit/Controls/CppApi/CoordinateConversion.qml

```diff
@@ -0,0 +1,128 @@
+import QtQuick 2.0
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import Esri.ArcGISRuntime.Toolkit.CppApi 100.2
+
+Rectangle {
+    id: coordinateConversionWindow
+    height: 800
+    width: 600
+
+    property bool getFromMapMode: false
+    property int buttonWidth: 100
+    property int spacingVal: 5
```

Comment:
I assume these are properties that we'd like to document what they do.  Since that's the case, they should probably have clearer names. Usually booleans don't start with `get`.  `spacingVal` should be spelled out `spacingValue`.  Also these pixel values need to have scaleFactor applied to them.  If the UI isn't fleshed out then we can spin these tweaks up as a separate issue.